rel/i18n/emqx_bridge_mqtt_connector_schema.hocon

```diff
@@ -33,9 +35,12 @@ egress_desc.label:
 """Egress Configs"""
 
 egress_pool_size.desc:
-"""Size of the pool of MQTT clients that will publish messages to the remote broker.<br/>
+"""~
+Size of the pool of MQTT clients that will publish messages to the remote broker.<br/>
 Each MQTT client will be assigned 'clientid' of the form '${clientid_prefix}:${bridge_name}:egress:${node}:${n}'
-where 'n' is the number of a client inside the pool."""
+where 'n' is the number of a client inside the pool.
+
+Does not take effect if static clientids are used.~"""
```

Comment:
```suggestion
This option does not take effect if `static_clientids` is not empty. When static client IDs is in use, pool size is always derived from the number of assigned IDs for each node.~"""
```

Reply:
It's not quite _derived_: it's _exactly_ the assigned clientids.